README.md

```diff
@@ -210,6 +210,10 @@ Copyright:
  © 2013-2014 RAD Game Tools and Valve Software.  
 License: [MIT](https://opensource.org/licenses/MIT)
 
+Files: `spng/*`  
+Copyright: © 2018-2023 Randy <randy408@protonmail.com>.  
```

Comment:
Sorry for the nitpick, but could you remove the email address here, please?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1708/commits/c58f643665001df2a2b46e4b63563632af082c81